apis/orgapi/orgapi.yaml

```diff
@@ -2035,21 +1339,72 @@ components:
                 rootOrgId: "0131630445447741440"
                 imgUrl: null
                 approvedDate: null
-                orgTypeId: null
                 homeUrl: null
                 isDefault: null
                 createdDate: 2020-12-01 09:52:46:962+0000
-                parentOrgId: null
                 createdBy: null
                 hashTagId: "0131630445447741440"
                 noOfMembers: null
-                status: 0 
+                status: 0
                 locationIds: [
                   "bc75cc99-9205-463e-a722-5326857838f8",
                   "aecac7ab-15e4-45c9-ac7b-d716444cd652",
                   "3f286755-a36f-45bb-9505-ffeeee5aa723"
                 ]
-      
+                organisationType: school
+                isTenant: true
+                orgLocation: null
+    orgapiOrgSearchResponseV2:
+      type: object
+      properties:
+        id:
+          type: string
+        ver:
+          type: string
+        ts:
+          type: string
+        responseCode:
+          type: string
+        result:
+          $ref: '#/components/schemas/orgapiorgapiApiResponse'
+      example:
+        id: api.org.search
+        ver: v2
+        ts: 2020-11-23 09:16:58:628+0000
+        params:
+          resmsgid:
+          msgid: ad7135b8-ef64-44bd-adaa-0b131a657689
+          err:
+          status: success
+          errmsg:
+        responseCode: OK
+        result:
+          response:
+            count: 1
+            content:
+              - keys: {}
+                channel": dikshaChannelNew
```

Comment:
@FsherinP @mohitga we cannot add 'Diksha' specific terms in Sunbird. please make this generic. I am also concerned about the specific numeric keys given in the examples. Is this right?

Reply:
@FsherinP remove the  DIKSHA from channel keyword
@Sunitashelar  numeric is message-id, should be fine.  @FsherinP please confirm once or change the message-id

Reply:
Did the changes